gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+This will enable managers to better understand their employee’s time commitment and professional benefits thereby increasing the likelihood of garnering approval to make InnerSource contributions.
+
+## Problem
+
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other. 
+It is difficult for employees to communicate to their management what commitment of time they will need to make to an InnerSource project and how they will benefit.  
+Managers have no uniform way of keeping track of or rewarding their employee’s involvement in InnerSource projects.
+
+## Story
+ 
+You’ve successfully created an InnerSource program at your company and have buy-in from senior management, middle-management and developers.  However, after nearly a year there have been few actual contributions made to any InnerSource projects outside of the teams that originally created them.  After interviewing all the parties involved the main sticking point seems to be that it is difficult to know what the exact level of commitment developers will be asked to make if they choose to get involved in an InnerSource project. There is also no uniform way of advertising what opportunities for contributors exist, what they’ll be asked to do and approximately how long it might take.  Managers are supportive and want their employees to participate but so far have lacked a way of documenting and accounting for their employees’ activities within InnerSource projects.  What can be done to improve this situation for all the parties involved (InnerSource project owners, potential contributors and development managers)? 
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+* It is difficult for managers to approximate what the time/effort commitment will be for employees taking on InnerSource project tasks.
+* The less insight a manager has into the effort and accomplishment of their employees in InnerSource projects, the more unlikely it is that he can reward their employees for their InnerSource work.
+* The transparency and reputational aspect of the code submissions within a large company should work to keep the code quality high for InnerSource contributions.
+* Individual employees may commit themselves to InnerSource project tasks without first getting approval from their manager or having the skills or time required to complete it.
+
+## Context
```

Comment:
Oooh, these look like **Forces** to me.  They are not going to change based off of the application of the pattern.

Reply:
We may have an opposite understanding of what is a **Force** and what is **Context**?  :man_shrugging:

Reply:
OK.  Will consider rewording so that these better describe forces.  These are potentially better expressed as forces that make people act in a certain way or take certain actions.

Reply:
Good point.  Yes.  The two divisions responsible for standing up the InnerSource portal and the Gig Marketplace were both under the umbrella of Enterprise Architecture.  This is important to point out as their central role is understood across the firm as one that should be providing centralized tooling to the rest of the organization.  It would be more difficult to imagine, for example, a Marketing or HR division garnering the funding or support to create such tools for the rest of the company.